Packs/CommonScripts/Scripts/HttpV2/HttpV2.py

```diff
@@ -0,0 +1,206 @@
+            raise DemistoException(err_msg, exception)
+        return res
+
+    @staticmethod
+    def _generic_error_handler(res):
+        status_code = res.status_code
+        if status_code == 400:
+            raise DemistoException(f"Bad request. Status code: {status_code}. Origin response from server: {res.text}")
+
+        if status_code == 401:
+            raise DemistoException(f"Unauthorized. Status code: {status_code}. Origin response from server: {res.text}")
+
+        if status_code == 403:
+            raise DemistoException(f"Invalid permissions. Status code: {status_code}. "
+                                   f"Origin response from server: {res.text}")
+
+        if status_code == 404:
+            raise DemistoException(f"The server has not found anything matching the request URI. Status code:"
+                                   f" {status_code}. Origin response from server: {res.text}")
+        if status_code == 500:
+            raise DemistoException(f"Internal server error. Status code: {status_code}."
+                                   f" Origin response from server: {res.text}")
+
+        if status_code == 502:
+            raise DemistoException(f"Bad gateway. Status code: {status_code}. Origin response from server: {res.text}")
+
+
+def create_headers(headers: Dict, request_content_type_header: str, response_content_type_header: str) \
+        -> Dict[str, str]:
+    """
+    Args:
+        headers: The headers the user insert.
+        request_content_type_header: The content type header.
+        response_content_type_header: The response type header.
+
+    Returns:
+        A dictionary of headers to send in the request.
+    """
+    if request_content_type_header in CONTENT_TYPE_MAPPER.keys():
+        request_content_type_header = CONTENT_TYPE_MAPPER[request_content_type_header]
+    if response_content_type_header in CONTENT_TYPE_MAPPER.keys():
+        response_content_type_header = CONTENT_TYPE_MAPPER[response_content_type_header]
+    if request_content_type_header and not headers.get('Content-Type'):
+        headers['Content-Type'] = request_content_type_header
+    if response_content_type_header and not headers.get('Accept'):
+        headers['Accept'] = response_content_type_header
+
+    return headers
+
+
+def get_parsed_response(res, resp_type: str) -> Dict:
+    try:
+        resp_type = resp_type.lower()
+        if resp_type == 'json':
+            res = res.json()
+        elif resp_type == 'text':
+            res = res.text
+        elif resp_type == 'xml':
+            res = json.loads(xml2json(res.content))
+        else:
+            res = res.text
+        return res
+    except ValueError as exception:
+        raise DemistoException('Failed to parse json object from response: {}'
+                               .format(res.content), exception)
+
+
+def get_status_list(status_list: list) -> List[int]:
```

Comment:
1. Let's change this name - we aren't getting the status list, we're changing its format if I understand correctly.
2. Let's add an explanation to the docstring

Reply:
done